modules/k8s-nmstate-troubleshooting-dns-disconnected-env.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-troubleshooting-node-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-dns-disconnected-env_{context}"]
+= Troubleshooting DNS connectivity issues in a disconnected environment
+
+If you experience DNS connectivity issues when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve the list of name servers for the domain `root-servers.net`.
+
+== Configuring the bind9 DNS named server  
+
+For a cluster configured to query a `bind9` DNS server, you can add the `root-servers.net` zone to a configuration file that contains at least one NS record.
+For example you can use the `/var/named/named.localhost` as a zone file that already matches this criteria.
+
+.Procedure
+
+. Add the `root-servers.net` zone at the end of the `/etc/named.conf` configuration file by running the following command:
++
+[source,terminal]
+----
+$ cat >> /etc/named.conf <<EOF
+zone "root-servers.net" IN {
+    	type master;
+    	file "named.localhost";
+};
+EOF
+----
+
+. Restart the `named` service by running the following command:
++
+[source,terminal]
+----
+$ systemctl restart named
+----
+
+. Confirm that the `root-servers.net` zone is present by running the following command:
++
+[source,terminal]
+----
+$ journalctl -u named|grep root-servers.net
+----
++
+.Example output
+[source,terminal]
+----
+Jul 03 15:16:26 rhel-8-10 bash[xxxx]: zone root-servers.net/IN: loaded serial 0
+Jul 03 15:16:26 rhel-8-10 named[xxxx]: zone root-servers.net/IN: loaded serial 0
+----
+
+. Verify that the DNS server can resolve the NS record for the `root-servers.net` domain by running the following command:
++
+[source,terminal]
+----
+$ host -t NS root-servers.net. 127.0.0.1
```

Comment:
Is this the default address for the root server? I suppose what I'm asking is where or how does the user get this information?

Reply:
Thanks for the review @JoeAldinger 
The backstory here. This was a bug that was resolved in OCP 4.12.42. It boils down to correcting a typo 
`kubernetes-nmstate` DNS check was configured to look up `root-server.net` while the correct value is  `root-servers.net` (spot the difference ;-))
I realise there are two addresses here, I'm not sure which you meant now but the IP address `127.0. 0.1` is the loopback or localhost address and is used by a computer to refer to itself.